tools/chainsync-loadtest/Cargo.toml

```diff
@@ -16,8 +16,7 @@ name = "chainsync-loadtest"
 clap = "=3.0.0-beta.2"
 actix = "=0.11.0-beta.2"
 parking_lot = "0.11.2"
-tracing = "0.1.13"
-tracing-subscriber = "0.2.4"
+tracing-subscriber = "0.3.9"
```

Comment:
We might want to re-export tracing crates form o11y, to have fewer versions in Cargo.tomls